qa/L0_java_memory_growth/Simple.java

```diff
@@ -394,6 +394,13 @@ output0, output1, new FloatPointer(input0_data), new FloatPointer(input1_data),
       } else {
         return false;
       }
+      if((memory_snapshots.get(index_90) / 1E6) >= 30){
```

Comment:
Not too familiar with java, but shouldn't these magic numbers governing constraints be a some kind of constants?

Reply:
Thanks! Addressed.